libs/SmartStore/SmartStore/Classes/SFSmartStoreInspectorViewController.m

```diff
@@ -28,7 +28,6 @@
 
 #import "SFSmartStoreInspectorViewController.h"
```

Comment:
But right now it looks like it knows of all the other windows and we don't want a SmartStore only window to be referenced in the SalesforceSDKCode library though.